gallery/visualtests/nxDrawer.js

```diff
@@ -0,0 +1,186 @@
+
+    it('looks right', async function() {
+      await getPage().setViewport(viewportSize);
+      await openDrawer(buttonId, drawerId);
+      await checkFullPageScreenshot();
+    });
+
+    it('passes a11y', async function() {
+      await getPage().setViewport(viewportSize);
+      await openDrawer(buttonId, drawerId);
+      await a11yTest();
+    });
+  });
+
+  describe('NxDrawer with Subtitle or Description Example', function() {
+    const browserSetup = setupBrowser('#/NxDrawerWithSubtitleOrDescriptionExample', false);
+    const { getPage, checkFullPageScreenshot, a11yTest } = browserSetup;
+    const openDrawer = openDrawerFromPage(browserSetup);
+
+    describe('with subtitle', function() {
+      const buttonId = 'nx-drawer-with-subtitle-open-button';
+      const drawerId = 'nx-drawer-with-subtitle';
+
+      it('looks right', async function() {
+        await getPage().setViewport(viewportSize);
+        await openDrawer(buttonId, drawerId);
+        await checkFullPageScreenshot();
+      });
+
+      it('passes a11y', async function() {
+        await getPage().setViewport(viewportSize);
+        await openDrawer(buttonId, drawerId);
+        await a11yTest();
+      });
+    });
+
+    describe('with description', function() {
+      const buttonId = 'nx-drawer-with-description-open-button';
+      const drawerId = 'nx-drawer-with-description';
+
+      it('looks right', async function() {
+        await getPage().setViewport(viewportSize);
+        await openDrawer(buttonId, drawerId);
+        await checkFullPageScreenshot();
+      });
+
+      it('passes a11y', async function() {
+        await getPage().setViewport(viewportSize);
+        await openDrawer(buttonId, drawerId);
+        await a11yTest();
+      });
+    });
+
+    describe('with both', function() {
+      const buttonId = 'nx-drawer-with-subtitle-and-description-open-button';
+      const drawerId = 'nx-drawer-with-subtitle-and-description';
+
+      it('looks right', async function() {
+        await getPage().setViewport(viewportSize);
+        await openDrawer(buttonId, drawerId);
+        await checkFullPageScreenshot();
+      });
+
+      it('passes a11y', async function() {
+        await getPage().setViewport(viewportSize);
+        await openDrawer(buttonId, drawerId);
+        await a11yTest();
+      });
+    });
+  });
+
+  describe('NxDrawer with Narrow Variant', function() {
+    const browserSetup = setupBrowser('#/NxDrawerVariantExample', false);
+    const { getPage, checkFullPageScreenshot, a11yTest } = browserSetup;
+    const openDrawer = openDrawerFromPage(browserSetup);
+
+    const buttonId = 'nx-drawer-variant-narrow-open-button';
+    const drawerId = 'nx-drawer-variant-narrow';
+
+    it('looks right', async function() {
+      await getPage().setViewport(viewportSize);
+      await openDrawer(buttonId, drawerId);
+      await checkFullPageScreenshot();
+    });
+
+    it('passes a11y', async function() {
```

Comment:
I don't think we need to check a11y again for the narrow variant.

Reply:
> I don't think we need to check a11y again for the narrow variant.

Fixed: 9c21e07f43dcbb21c13cb39d30127aadff0b9b4f